build/auth.md

```diff
@@ -80,7 +80,7 @@ into their respective files in `$HOME`.
 1.  Execute the build:
 
     ```shell
-    kubectl apply --filename secret.yaml serviceaccount.yaml build.yaml
+    kubectl apply -f secret.yaml -f serviceaccount.yaml -f build.yaml
```

Comment:
@gyliu513  please use the long version of the command, --filename

Reply:
Done, thanks @samodell